woocommerce/compatibility/class-sv-wc-order-compatibility.php

```diff
@@ -522,6 +524,200 @@ public static function get_edit_order_url( \WC_Order $order ) {
 	}
 
 
+	/**
+	 * Gets the admin screen ID for orders.
+	 *
+	 * This method detects the expected orders screen ID according to HPOS availability.
+	 * `shop_order` as a registered post type as the screen ID is no longer used when HPOS is active.
+	 *
+	 * @since x.y.z
+	 *
+	 * @return string
+	 */
+	public static function get_screen_id( ) : string {
```

Comment:
The name should be `get_admin_order_screen_id()` or `get_order_screen_id()` for better readability.

Reply:
Aren't screens only for admin contexts in WP? I'd vote for the latter.